.golangci.json

```diff
@@ -42,7 +45,11 @@
     "default-severity": "error",
     "rules": [
       {
-        "text": "(ST\\d{4}|S\\d{4}|SA1019)",
+        "linters": ["stylecheck", "gosimple", "unused", "errcheck", "forcetypeassert"],
```

Comment:
Figured out that I could be a bit better about naming linters that should be warnings, and added our new options to it (`errcheck` and `forcetypeassert` are currently disabled by global config, but this will make them warnings when we do turn them on)

Reply:
:+1: